sourcegraph/sourcegraph
# storm: include temporary settings to jscontext

## Context

- Follow-up on https://github.com/sourcegraph/sourcegraph/pull/47010
- Adds server preloading for temporary settings that we load on every page.
- We use preloaded data on the client by hydrating the Apollo Client cache before Apollo Client init. Nothing changes in the client. Telling the `TemporarySettingsBackend` to use the `cache-first` policy is enough to eliminate the HTTP request.

## Test plan

1. Ensure that `experimentalFeatures.enableStorm === true`
2. `sg start`
3. Open any web application page.
4. Notice in the network tab that no API requests are sent to load `temporarySettings`.
